tests/integration/test_schema_cache.py

```diff
@@ -0,0 +1,26 @@
+import os
+
+from app import cache, settings
+from tests.integration.create_token import create_token
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class TestApplicationVariables(IntegrationTestCase):
+
+    def test_schema_is_cached(self):
+        with self.application.app_context():
+
+            self.assertEqual(len(cache.cache._cache), 0)
```

Comment:
Hmm, this is a bit weird. Can we import from app.cache to save drilling down this far?